### ISSUE TITLE
Objects are not valid as a React child dataframe contains nested objects

### ISSUE THREAD

**Issue by ml054** [grafana/grafana#47531]
**What happened**:
In 8.4.5 version I get following error when rendering: 
`Objects are not valid as a React child (found: object with keys {@projection, @change-vector, @flags, @id, @index-score, @last-modified}). If you meant to render a collection of children, use an array instead.`

![image](https://user-images.githubusercontent.com/1250431/162507830-1f356490-7de3-4571-93af-033e5f423fc8.png)

**What you expected to happen**:
See results in panel with nested objects. Like this:
![image](https://user-images.githubusercontent.com/1250431/162504791-1d9f0153-8db1-4c58-bca2-b128ecb75bdd.png)

**How to reproduce it (as minimally and precisely as possible)**:

Use nested object literals in dataframe. 

Minimal dataframe:
```
export function responseToDataFrame(response: QueryResponseDto): DataFrame[] {
  const dataFrame = toDataFrame([
    {
      a: {
        b: true,
      },
    },
  ]);

  return [dataFrame];
}
```

**Anything else we need to know?**:

**Environment**:
- Grafana version: 8.4.5
- Data source type & version: ravendb-datasource
- OS Grafana is installed on: windows
- User OS & Browser: chrome 1000
- Grafana plugins: ravendb-datasource
- Others:

**Comment by zuchka** [comment]
Thanks for creating this issue, @ml054 

Can you reproduce this issue without using a third-party data source plugin like ravendb-datasource? thx 👍

**Comment by ml054** [comment]
Ok it seems to be fixed as a part of: https://github.com/grafana/grafana/commit/5a87d12e8c321e6680e187b2de0e80510fbfe672#diff-9832424a4de5ea17c5d6ad647893b61a4b60bb2ff273be681be649b2c3ed51c6R32 
https://github.com/grafana/grafana/pull/47074

Error was here:
https://github.com/grafana/grafana/blame/a571fcda6ea63024be52ed7f0f3e8cd91b401a01/packages/grafana-ui/src/components/Table/JSONViewCell.tsx

![image](https://user-images.githubusercontent.com/1250431/162897832-becbeb1f-0ec4-4716-97b9-a8aaaf3273a7.png)

in 1 - we have object literal
in 2 - we pass object literal as child - which is wrong

**Comment by ml054** [comment]
Any option to backport that change to 8.4.x branch? Yesterday 8.4.6 was released and it is also broken version.

**Comment by zuchka** [comment]
@connorlindsey any thoughts on backporting your PR? Or is it better to sit tight a bit longer until 8.5.0 is out of beta.

@ml054 8.5.0 GA should get released ~4/21

**Comment by connorlindsey** [comment]
@zuchka The PR could definitely be backported if we're going to cut an 8.4.7 release

**Comment by ml054** [comment]
I'd even apply oneline fix: 
https://github.com/grafana/grafana/blame/a571fcda6ea63024be52ed7f0f3e8cd91b401a01/packages/grafana-ui/src/components/Table/JSONViewCell.tsx#L32
and change: 
```
{!link && <div className={tableStyles.cellText}>{value}</div>}
```
to 
```
{!link && <div className={tableStyles.cellText}>{displayValue}</div>}
```

**Comment by ashharrison90** [comment]
@ml054 looks like we never did backport this to 8.4.x unfortunately. however given the fix has been released in 8.5.0, i'm going to close this now. hope that makes sense! 😄

**Comment by ml054** [comment]
Yes we also recommend to use 8.5+ in RavenDB and it works. Thanks.
